project/funcoes.class.js

```diff
@@ -0,0 +1,29 @@
+// Classe para chamar o Json.
+function json(){
+	var qtd;
+	var retorno;
+
+	// Resgatar valores.
+	json.prototype.resgatarValores = function(){
```

Comment:
Why did you choose to make use of prototype here?
Anyway, it's not recommended (and it kind doesn't make sense) to define prototype methods inside a constructor function. Take a look at the first answer for clarification: http://stackoverflow.com/questions/2784844/setting-javascript-prototype-function-within-object-class-declaration